src/chainstate/coordinator/tests.rs

```diff
@@ -3236,6 +3236,187 @@ fn test_stx_transfer_btc_ops() {
     }
 }
 
+#[test]
+fn test_sbtc_peg_in_btc_op() {
+    let path = "/tmp/stacks-blockchain-sbtc-peg-in-btc-ops";
+    let _r = std::fs::remove_dir_all(path);
```

Comment:
Can you just `.unwrap()` this so the test fails if the filesystem is inaccessible for this test?

Reply:
I can't simply unwrap, since that would cause the test to panic if the directory do not exist. I'm not a super fan of this pattern of ignoring results either but it's repeated in most test cases in this module so I'd recommend any changes to this pattern to be solved as a separate PR.